listenbrainz/webserver/views/metadata_api.py

```diff
@@ -16,6 +16,9 @@
 
 metadata_bp = Blueprint('metadata', __name__)
 
+#: The maximum length of the query permitted for a mapping search
+MAX_MAPPING_QUERY_LENGTH = 500
```

Comment:
That seems rather permissive still. Do you have an idea of what a typical query length is?

Reply:
The typical query length is likely less than a 100 characters, 95%+ of the data in that range. There are about 600k recordings in the 100-250 characters range. I think I'll lower the limit to 250, there are just 10-15k recordings out of 24 million there.

Reply:
The typical query length is likely less than a 100 characters, 95%+ of the data in that range. There are about 600k recordings in the 100-250 characters range. I think I'll lower the limit to 250, there are just 10-15k recordings out of 24 million there.